### PR TITLE
docs: Add commas to infer.mle.fixed_poi_fit docstring for clarity

### LINKED ISSUE #1234: Add commas around constrained maximal likelihood in docstring for clarity
# Description

In PR #905 the docstring for [`pyhf.infer.mle.fixed_poi_fit`](https://scikit-hep.org/pyhf/_generated/pyhf.infer.mle.fixed_poi_fit.html) was amended, but the lines

https://github.com/scikit-hep/pyhf/blob/fd7930cce36cbc3a2d0ee1828f060d7382129579/src/pyhf/infer/mle.py#L134-L135

are missing commas around the likelihood, making it difficult to read

missing_commas

It should read

```
,:math:`L\left(\mu, \hat{\hat{\boldsymbol{\theta}}}\right)`, in the profile 
```

### PR DESCRIPTION
**Pull request by 1nF0rmed**
# Description

Resolves #1234 

As mentioned in the issue, there was a readability problem with the following lines:
https://github.com/scikit-hep/pyhf/blob/fd7930cce36cbc3a2d0ee1828f060d7382129579/src/pyhf/infer/mle.py#L134-L135

I have added the comma around the constrained maximal likelihood as mentioned in the issue.

ReadTheDocs build: https://pyhf.readthedocs.io/en/docs-build-docs-docfix-1nformed/_generated/pyhf.infer.mle.fixed_poi_fit.html

# Checklist Before Requesting Reviewer

- [x] Tests are passing
- [x] "WIP" removed from the title of the pull request
- [x] Selected an Assignee for the PR to be responsible for the log summary

# Before Merging

For the PR Assignees:

- [x] Summarize commit messages into a comprehensive review of the PR

```
* Add commas around constrained maximal likelihood for reading clarity
* Add Pradyumna to list of contributors
```

### CODE REVIEW

**Comment by 1nF0rmed** [comment]
@matthewfeickert Can you please review this PR?

**Comment by matthewfeickert** [comment]
Thanks for your PR @1nF0rmed &mdash; your contributions are very welcome! Would you please add your name to `docs/contributors.rst`?

File: src/pyhf/infer/mle.py
Comment (matthewfeickert) [inline src/pyhf/infer/mle.py:135]: When I build the docs locally the first comma that has no space between it and the `:math:` breaks Sphinx syntax, and if I then put a space `, :math:` the render looks a bit strange. Can you remove the first comma from line `135` 
```suggestion
    :math:`L\left(\mu, \hat{\hat{\boldsymbol{\theta}}}\right)`, in the profile
```
and put it at the end of line `134` instead (just after the :math:`\mu`)?

**Review by matthewfeickert — CHANGES_REQUESTED** [review]
Thanks @1nF0rmed. Just one small change to make sure the docs build looks okay.

**Review by matthewfeickert — APPROVED** [review]
LGTM. :+1: Thanks very much @1nF0rmed!

There's some CI checks that only trigger when a core dev initiates them, which I'll run now, but after those finish I'll go ahead and merge.

**Pull request merged**
